app/models/casa_admin.rb

```diff
@@ -46,7 +46,7 @@ def change_to_supervisor!
 #  created_at                  :datetime         not null
 #  updated_at                  :datetime         not null
 #  casa_org_id                 :bigint           not null
-#  invited_by_id               :bigint
+#  invited_by_id               :integer
```

Comment:
these extraneous comment changes are not ideal but they are common. Don't worry about them for now.

Reply:
thats weird.. i never touched any model files